### PR TITLE
Pull in user data from editable source that can be updated via pr

### PR DESCRIPTION
**Pull request by ajdeleon**
Issue, Bugfix or New Feature:
Related  issue: #6 

## Description
I created a new page called members. This could be changed to "Users" or something similar. This maps through an object that is held in data/data.js to populate the members. 

## Tasks
- [x] Pull in data from a data source than can easily be updated by users via a pull request

## :warning:  Warning :warning:
No warnings for this one as it was a pretty straightforward addition

## Next steps
I want to now work on making individual user pages that will still use the same data.js as their source of data. These will by dynamically rendered and have pretty urls e.g. phillyshowcase.com/ajdeleon

### CODE REVIEW

File: showcase/src/data/data.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export default [
```
Comment (adamzev) [inline showcase/src/data/data.js:1]: Should each user have their own variable and then add that variable to this list? What will be the cleanest when this list grows very large? This is a separate issue, but I'd like a commit hook that checks that the structure is valid.

File: showcase/src/data/data.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  {
+    image: 'http://via.placeholder.com/180x320',
+    username: 'test user',
```
Comment (adamzev) [inline showcase/src/data/data.js:4]: Put the username first and maybe rename this to be displayName as I don't usually think of usernames as allowing spaces.

Comment (adamzev) [reply showcase/src/data/data.js:4]: Or both a username and display name?

File: showcase/src/data/data.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      'project5',
+      'project6',
+    ],
+  },
+  {
+    image: 'http://via.placeholder.com/180x320',
+    username: 'test user 2',
+    tagline: 'Came for the pizza',
+    bio: 'I am a human being. No really, I am totally not a computer',
+    projects: ['project1', 'project2', 'project3', 'project4'],
```
Comment (adamzev) [inline showcase/src/data/data.js:21]: This is good for now, but of course, projects will need to be their own structure. I'm curious/concerned how large each individual's json data structure will need to be. We are at 30 lines with about half the data we'll eventually need. Maybe each user can create their own file by copying a template and export that variable and include it in this structure? They'd have to edit two files but their edit in the shared file would be smaller. If we need to change the data structure in the future we'll have to have a script read it in and create new files with the changes.

**Review by adamzev — APPROVED** [review]
Thanks! I had a few thoughts seeing it done out but this looks great!

File: showcase/src/data/data.js (hunk 1 above)
Comment (ajdeleon) [reply showcase/src/data/data.js:1]: Yeah good point. That is a better way of doing it. Will work on it

File: showcase/src/data/data.js (hunk 2 above)
Comment (ajdeleon) [reply showcase/src/data/data.js:4]: Oh yeah display name would be cool. Easy to add too

File: showcase/src/data/data.js (hunk 3 above)
Comment (ajdeleon) [reply showcase/src/data/data.js:21]: Yep that is a better idea. I will make the change when I have time. It will be much more scalable

**Pull request merged**